### PR TITLE
Refactor frontend exports

### PR DESCRIPTION
**Pull request by benrudolph**
@NoahCarnahan @snopoke this is gonna be a bit of marathon. this successfully reroutes exports based on the `NEW_EXPORT` flag. it also adds enough functionality to at least have the page render without errors on the new export. realize there's still a long way to go/messy things, but want to try and do this as piece meal as possible. note this PR will get significantly shorter once https://github.com/dimagi/commcare-hq/pull/10136 is merged. can be read :fish: 

cc: @millerdev @emord

### CODE REVIEW

File: corehq/apps/export/static/export/spec/data/export_instances.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                        "value": "choice2"
+                    }]
+                },
+                "selected": false,
+                "label": "question3",
+                "doc_type": "ExportColumn"
+            }],
+            "doc_type": "TableConfiguration"
+        }],
+        "name": null,
```
Comment (dimagimon) [inline corehq/apps/export/static/export/spec/data/export_instances.js]: Duplicate key 'name'.

File: corehq/couchapps/schemas_by_xmlns_or_case_type/views/view/map.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+function(doc) {
```
Comment (dimagimon) [inline corehq/couchapps/schemas_by_xmlns_or_case_type/views/view/map.js]: Missing name in function declaration.

**Comment by snopoke** [comment]
```
======================================================================
ERROR: test_generate_instance_from_schema (corehq.apps.export.tests.test_export_instance.TestExportInstanceGeneration)
Only questions that are in the main table and of the same version should be shown
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/export/tests/test_export_instance.py", line 54, in test_generate_instance_from_schema
    self.app_id
  File "/mnt/corehq/apps/export/models/new.py", line 194, in generate_instance_from_schema
    type=schema.type
AttributeError: 'ExportDataSchema' object has no attribute 'type'
```

**Comment by NoahCarnahan** [comment]
nice!

**Comment by benrudolph** [comment]
oh yikes: `The job exceeded the maxmimum time limit for jobs, and has been terminated.`

**Comment by snopoke** [comment]
Not mergable

File: corehq/apps/export/templates/export/new_customize_export.html [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+    <script src="{% static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
+    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
+    <script src="{% static 'jquery-ui-1.8.23-legacy/jquery-ui.min.js' %}"></script>
+    <script src="{% static 'export/js/models.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %}{{ block.super }}
+    <script>
+        $(function () {
+            var translations = {
```
Comment (snopoke) [inline corehq/apps/export/templates/export/new_customize_export.html]: Is this used somewhere?

Comment (benrudolph) [inline corehq/apps/export/templates/export/new_customize_export.html]: No but this file is slowly going to get refactored. This is all stuff from
the old customize_export.html. I have another branch where I'm working on
stuff like this

**Pull request merged**